services/galley/src/Galley/API/BackgroundProcesses.hs

```diff
@@ -0,0 +1,127 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2022 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module Galley.API.BackgroundProcesses where
+
+import Cassandra (ClientState, Consistency (LocalQuorum), Page (hasMore, nextPage, result), paginate, paramsP)
+import Control.Lens hiding (Getter, Setter, (.=))
+import Data.List.NonEmpty qualified as N
+import Data.Map qualified as Map
+import Data.Qualified
+import Data.Range
+import Galley.API.Federation (insertIntoMap)
+import Galley.API.Util
+import Galley.Cassandra.Queries qualified as Q
+import Galley.Cassandra.Store (embedClient)
+import Galley.Data.Conversation (convTeam)
+import Galley.Data.Conversation.Types
+import Galley.Data.Scope (Scope (ReusableCode))
+import Galley.Effects
+import Galley.Effects.CodeStore qualified as E
+import Galley.Effects.ConversationStore
+import Galley.Effects.ConversationStore qualified as E
+import Galley.Effects.MemberStore qualified as E
+import Galley.Effects.TeamStore qualified as E
+import Galley.Types.Conversations.Members (RemoteMember (RemoteMember, rmId))
+import Galley.Types.UserList
+import Imports hiding (head)
+import Polysemy
+import Polysemy.Input
+import Wire.API.Conversation hiding (Member)
+
+-- | Remove all conversations and members from the local domain. This must only be run in a background process
+unsafeRemoveRemoteMembersFromLocalConversation ::
```

Comment:
why is it `unsafe`?  because it's not fit for HTTP?  then i'd call it `bg` instead.